### PR TITLE
[NRM 330] Enabling screen reader to read header and paragraph text

### PR DESCRIPTION
**Pull request by Adesh-Ramchurn**
## What?
Paragraph and header not announced by screen reader on the following page pv-under-age. See following ticket [NRM-330](https://collaboration.homeoffice.gov.uk/jira/browse/NRM-330)
## Why?
This could be due to the radio button element set up on the HoF framework. To fix this issue, we updated the pv-under-age HTML page and hard coded the elements.
## How?
A div element was added to include a tab index which was set to 0. This enabled the tab function to pick up the header and the paragraph.
## Testing?
Voice Over was used as a screen reader to test whether the header and the paragraph are being picked up. 
## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [x] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

File: apps/nrm/views/pv-under-age.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {{<partials-page}}
   {{$page-content}}
-    {{#markdown}}pv-under-age{{/markdown}}
+  <div  tabindex="0">
+    <h1 class="govuk-heading-l">Is the potential victim under 18?</h1>
+    <p>If you’re not sure of the potential victim’s age, you’ll need to refer them to the relevant local authority for an age assessment. They will continue to be treated as a child until a decision on their age is made.</p>
+    <p>If they’re under 18 or you’re not sure, you must explain to them how organisations such as the Home Office will use their information.</p>
```
Comment (tobyschaefferHO) [inline apps/nrm/views/pv-under-age.html:6]: would you replace these with entries places in the route under pages.json?  It'll be more consistent with our use of the translation library.

File: apps/nrm/views/pv-under-age.html [hunk 2]
```diff
@@ -1,6 +1,10 @@
 {{<partials-page}}
   {{$page-content}}
-    {{#markdown}}pv-under-age{{/markdown}}
+  <div  tabindex="0">
```
Comment (tobyschaefferHO) [inline apps/nrm/views/pv-under-age.html:3]: extra space here should be deleted.  
```suggestion
  <div tabindex="0">
```

**Review by tobyschaefferHO — CHANGES_REQUESTED** [review]
Update works great - wouldn't have thought of that.  William and Rhodine have expressed they want us to place text within the pages.json file, and to use those entries within our html templates - would you mind doing that?  Once you've got that i'll be happy to approve.  thanks again - nice work!

**Review by william-gu-hof — APPROVED** [review]

**Review by tobyschaefferHO — APPROVED** [review]
code looks good 👍

**Comment by tobyschaefferHO** [comment]
Merging as branch test approval received.

**Pull request merged**